fastly/fixtures/tokens/create_and_bulk_delete.yaml

```diff
@@ -47,32 +50,35 @@ interactions:
       X-Cache-Hits:
       - 0, 0
       X-Served-By:
-      - cache-control-slwdc9035-CONTROL-SLWDC, cache-man4150-MAN
+      - cache-control-slwdc9035-CONTROL-SLWDC, cache-man4144-MAN
       X-Timer:
-      - S1631722256.303142,VS0,VE306
+      - S1632137078.768877,VS0,VE400
     status: 200 OK
     code: 200
     duration: ""
 - request:
-    body: name=my-test-token-1&password=foobar&scope=global&username=testing%40fastly.com
+    body: name=my-test-token-1&password=foobar&scope=global&services%5B%5D=5wHdzYhbILBcvU2aAjH8so&services%5B%5D=6besTz8jIUQudhhr4vLEzt&username=testing%40fastly.com
     form:
       name:
       - my-test-token-1
       password:
       - foobar
       scope:
       - global
+      services[]:
+      - 5wHdzYhbILBcvU2aAjH8so
+      - 6besTz8jIUQudhhr4vLEzt
       username:
       - testing@fastly.com
     headers:
       Content-Type:
       - application/x-www-form-urlencoded
       User-Agent:
-      - FastlyGo/3.11.0 (+github.com/fastly/go-fastly; go1.17)
+      - FastlyGo/3.12.0 (+github.com/fastly/go-fastly; go1.17)
     url: https://api.fastly.com/tokens
     method: POST
   response:
-    body: '{"id":"67WnFZog4TztGgdNfc9mxX","name":"my-test-token-1","user_id":"52LiGaZmbWl4D5xv4tcxON","customer_id":"51MumwLiSJyFTWhtbByYgR","service_id":null,"expires_at":null,"created_at":"2021-09-15T16:10:56Z","updated_at":"2021-09-15T16:10:56Z","scope":"global","services":[],"access_token":"XXXXXXXXXXXXXXXXXXXXXX"}'
+    body: '{"id":"27DePetT24oLN9xdXRMkdo","name":"my-test-token-1","user_id":"52LiGaZmbWl4D5xv4tcxON","customer_id":"51MumwLiSJyFTWhtbByYgR","service_id":"6besTz8jIUQudhhr4vLEzt","expires_at":null,"created_at":"2021-09-20T11:24:38Z","updated_at":"2021-09-20T11:24:38Z","scope":"global","services":["5wHdzYhbILBcvU2aAjH8so","6besTz8jIUQudhhr4vLEzt"],"access_token":"XXXXXXXXXXXXXXXXXXXXXX"}'
```

Comment:
Notice the API response now correctly includes a value for the `services` field.